gvfs.md

```diff
@@ -160,18 +161,80 @@ The following fields are all attributes within the main "data" object for this f
+\- passenger_capacity | Optional | The amount of people than can travel at once on this vehicle
+\- enclosed | Optional | 1/0 boolean - whether the vehicle contains an enclosed environment that isolates the passenger(s) from the outside elements
+
+Example:
+
+```json
+{
+  "last_updated": 1434054678,
+  "ttl": 12345,
+  "data": {
+    "system_id": "inboard",
+    "language": "en",
+    "name": "Inboard",
+    "short_name": "ib",
+    "url": "https://www.inboardtechnology.com/",
+    "timezone": "America/Los_Angeles",
+    "vehicle_types": [
+      {
+        "vehicle_type_id": "m1",
+        "name": "M1 Electric Skateboard",
+        "description": "Experience the most advanced e-board deck ever developed. A custom designed composite deck that combines an inverted 3D Poplar wood core with full sandwich PU sidewalls and wrapped in the same fiberglass top sheet found in premium snowboards and skis. Integrated electronics, reinforced truck mounting points, and resilient TPU nose and tail bumpers at the rails yield a remarkably light and durable platform optimized for a more responsive ride.",
+        "vehicle_url": "https://www.inboardtechnology.com/products/m1-electric-skateboard",
+        "image_small": "http://cdn.shopify.com/s/files/1/1136/4406/files/Skateboard_x150.png?11016692238788873189",
+        "image_large": "http://cdn.shopify.com/s/files/1/1136/4406/products/m1_pdp_fullBleed_6_9e6614b5-c3e4-4dd1-b18b-fd51451d0c17_1600x.jpg?v=1535665708",
+        "propulsion_type": "ELECTRIC",
+        "maximum_cruising_speed": 25,
+        "range_with_full_energy_potential": 11,
+        "vehicle_weight": 6.57,
+        "power_output": 200,
+        "passenger_capacity": 1,
+        "enclosed": 0
+      }, {
+        "vehicle_type_id": "tg",
+        "name": "The Glider Scooter",
+        "description": "The ultimate scooter experience, crafted with premium components, top-quality materials, and relentless attention to detail. We engineered our Glider from the ground up, with the ride experience as our ultimate focus at every step of the way; providing a smooth, safe, and insanely fun ride, every time.",
+        "vehicle_url": "https://www.inboardtechnology.com/products/the-glider",
+        "image_small": "http://cdn.shopify.com/s/files/1/1136/4406/files/glider_nav_1_x150.png?11016692238788873189",
+        "image_large": "http://cdn.shopify.com/s/files/1/1136/4406/products/glider_pdp_4_super_duper_extended_2_de0ab82e-e603-4455-8395-dd6ec3b28cbb_375x375_crop_center.png?v=1542056407,%20//cdn.shopify.com/s/files/1/1136/4406/products/glider_pdp_4_super_duper_extended_2_de0ab82e-e603-4455-8395-dd6ec3b28cbb_750x750_crop_center.png?v=1542056407%202x",
+        "propulsion_type": "ELECTRIC",
+        "maximum_cruising_speed": 25,
+        "range_with_full_energy_potential": 19.3,
+        "vehicle_weight": 15.88,
+        "power_output": 350,
+        "passenger_capacity": 1,
+        "enclosed": 0
```

Comment:
Maybe add a field for if and what kind of driver license is needed according to local rules.

Reply:
Is there some data standard that we can reference to add this data?

Reply:
Within the European Union we do have a standardised classification https://en.wikipedia.org/wiki/European_driving_licence, I don't know if there is any data standard for this.